### ISSUE TITLE
Bring back option to remove auto uploaded files on Android

### ISSUE THREAD

**Issue by mayonezo** [nextcloud/android#14204]
Please bring back the option to remove auto-uploaded files on android!

This change breakes many workflows.

### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by thejohnha** [comment]
Which android version are you on? I'm having this issue also. Additionally, certain auto-upload folders do work, but others do not. And on top of that, if I manually go to upload certain files on my phone, it's allowed, but certain other files/folders are not (and the files are not even displayed in the file picker). If I use a file explorer app, I can clearly see that the file or folder exists, but the file picker does not display them. Very strange. I wonder if it's a permissions issue, or related to this new message I saw:
Auto upload behaviour changed
Due to new restrictions imposed by Google, the auto upload feature will no longer be able to automatically remove uploaded files.

**Comment by mayonezo** [comment]
Im on 3.30.6, where the change was introduced.

**Comment by mwildbolz** [comment]
would also be very interested to get back the original behaviour. I'm on app version 3.30.6

At the moment, I'm not able to have a good overview which files are uploaded and which ones not (my "workflow" doesnt work).
Main problem is that I cannot be sure at the moment, that all files are uploaded correctly. Maybe somenone of you has an idea, how I can check this?

But also:
Freeing the space of backup'ed photos is in my opinion a main function which is needed in an app of this kind (like Google Photos, etc.)?!?

**Comment by HPPinata** [comment]
This is due to the Google Team in charge of the Play-Store not allowing the Nextcloud App to access all files.
They weren't allowed to publish ANY updates until they removed that permission, see https://github.com/nextcloud/android/pull/14099. Instead they have to use the Android media Subsystem that can still access files everywhere, but is limited to only media files and read only access at that.

The only way to make things work again is to uninstall the Play-Store Version, install the F-Droid alternative AppStore https://f-droid.org/ and then install the Nextcloud App from there.
In that Version full storage access isn't (arbitrarily at best, maliciously at worst) forbidden by Google.

**Comment by HPPinata** [comment]
This is an existing issue: See https://github.com/nextcloud/android/issues/14199
This should be closed to keep discussions all in one place.

**Comment by mayonezo** [comment]
F-Droid version is 3.30.5 for me. #14199 is a different issue, although probably due to the same permission change mandated by Google.

> This is due to the Google Team in charge of the Play-Store not allowing the Nextcloud App to access all files. They weren't allowed to publish ANY updates until they removed that permission, see [#14099](https://github.com/nextcloud/android/pull/14099). Instead they have to use the Android media Subsystem that can still access files everywhere, but is limited to only media files and read only access at that.
> 
> The only way to make things work again is to uninstall the Play-Store Version, install the F-Droid alternative AppStore https://f-droid.org/ and then install the Nextcloud App from there. In that Version full storage access isn't (arbitrarily at best, maliciously at worst) forbidden by Google.

**Comment by HPPinata** [comment]
![Image](https://github.com/user-attachments/assets/a0501628-cb00-4c97-b528-ebeddec366b5)

Anyways F-Droid works better than the gplay build

**Comment by HPPinata** [comment]
Though you are right I think the auto removal was killed as a side effect.

**Comment by mwildbolz** [comment]
just installed F-Droid version - now its working again. :-)
Thanks @HPPinata for the hint, haven't thought about this.

But, in general, I think it would be fine to get the original function back also on the official Google-Playstore way, because I think many people wouldnt like to use apps from sources like F-Droid...

**Comment by HPPinata** [comment]
> just installed F-Droid version - now its working again. :-) Thanks [@HPPinata](https://github.com/HPPinata) for the hint, haven't thought about this.
> 
> But, in general, I think it would be fine to get the original function back also on the official Google-Playstore way, because I think many people wouldnt like to use apps from sources like F-Droid...

Absolutely. This is at best a workaround and ideally Google would just stop being ヘ⁠（⁠。⁠□⁠°⁠）⁠ヘ and allow an app meant for uploading files access to the file system...

**Comment by dpoetzsch** [comment]
[heise.de reports that google gave in](https://www.heise.de/en/news/Google-gives-in-Nextcloud-gets-back-access-rights-for-Android-app-10388071.html) and nextcloud will be able to get full file access again. It states

>  There will soon be a corresponding update for the Nextcloud app.

This was a month ago. Any updates on this issue?

**Comment by Shentoza** [comment]
Bump
